js/script.js

```diff
@@ -26,6 +27,63 @@ const createTask = (taskTitle)=>{
 
 const deleteHandler = (taskId) => {
     tasks = tasks.filter((task) => task.id !== taskId);
+    if (currentTaskId === taskId) {
+        currentTaskId = null;
+    }
+    renderTasks();
+};
+
+const editHandler = (taskId, currentTitleElement, editButton) => {
+    if (currentTaskId !== null && currentTaskId !== taskId) {
+        cancelEdit();
+    }
+
+    currentTaskId = taskId;
+
+    const taskIndex = tasks.findIndex((task) => task.id === taskId);
+    const currentTask = tasks[taskIndex];
+    const newId = new Date().getTime();
+    console.log(taskId);
```

Comment:
remove this console.log

Reply:
@aka-wtag bhai I have removed unwanted lines and executed prettier to format the codes. Please can you review once again.
Thank you